ci/baseimage.Dockerfile

```diff
@@ -0,0 +1,58 @@
+#FROM nvcr.io/nvidia/nvhpc:22.9-devel-cuda_multi-ubuntu22.04 as builder
```

Comment:
this can be removed

Reply:
I've squashed the commits in https://github.com/electronic-structure/SIRIUS/tree/feature-ci-squashed. Shall we force push it to this branch in order to have fewer commits?

Reply:
I removed the commented line.

I can add the amdgpu_target ARG to the ROCm container and build with a single architecture. This doesn't matter because we only build.